torch/jit/_monkeytype_config.py

```diff
@@ -82,7 +101,9 @@ def consolidate_types(self, qualified_name: str) -> Dict:
                         _all_type += _type.__name__ + ','
                 _all_type = _all_type.lstrip(" ")  # Remove any trailing spaces
 
-                if len(types) > 1:
+                if len(types) == 2 and 'NoneType' in _all_type:
```

Comment:
Please add a TODO here to make sure we remove this hacky when `Union` support lands.

CC @ansley, PDT is much less useful without Union.

Reply:
Done